tests/sub_group_mask/sub_group_mask_common.h

```diff
@@ -0,0 +1,142 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Extension Conformance Test
+//
+//  Provides common methods for sub_group_mask tests
+//
+*******************************************************************************/
+
+#ifndef __SYCLCTS_TESTS_SUB_GROUP_MASK_COMMON_H
+#define __SYCLCTS_TESTS_SUB_GROUP_MASK_COMMON_H
+
+#include "../common/common.h"
+#include "../common/type_coverage.h"
+
+#ifdef SYCL_EXT_ONEAPI_SUB_GROUP_MASK
+namespace {
+
+static const auto types =
+    named_type_pack<char, signed char, unsigned char, short, unsigned short,
+                    int, unsigned int, long, unsigned long, long long,
+                    unsigned long long>{
+        "char",           "signed char", "unsigned char",     "short",
+        "unsigned short", "int",         "unsigned int",      "long",
+        "unsigned long",  "long long",   "unsigned long long"};
```

Comment:
I suggest we reduce the list of tested types by default and test the rest in "full mode".
Something like this: https://github.com/KhronosGroup/SYCL-CTS/pull/198/files#diff-4461c617ceae0c7e0206622aacdf555aedd62eb129c2efb74c84fa1567bcbe0d